### DISCUSSION TITLE
[Question]: What are the google domains prompted when I chat with gemini free api ?

### DISCUSSION THREAD

**Discussion by delta-whiplash** [danny-avila/LibreChat#2560] [Q&A]
### What is your question?

Hello because I live in france I have to route my librechat traffic to my VPN gateway on my firewall 
I found this domains and create a filtring rule to redirect the traffic on the good gateway 
![image](https://github.com/danny-avila/LibreChat/assets/39416418/971c1393-a079-44a3-b1fc-d84c92d3c774)

### More Details

But I have some random issues did I forgot one domain ?
![image](https://github.com/danny-avila/LibreChat/assets/39416418/87a88ffe-90b9-4e67-bece-c2f882094816)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Check your logs (in the ./logs directory of project root if using docker). Can't really help you much with these screenshots

**Reply by danny-avila** [reply]
Also, I recommend using Vertex AI which should be available in your region: https://docs.librechat.ai/install/configuration/ai_setup.html#vertex-ai

**Reply by delta-whiplash** [reply]
sry my logs didn't paste 


2024-04-27T13:59:42.560Z error: [handleAbortError] AI response error; aborting request: [GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1be... [truncated]
2024-04-27T13:59:42.561Z warn: AI Response error for conversation 5893bf6e-b3ea-4de6-9473-8aaacb390e21 likely caused by Google censor/filter
2024-04-27T13:59:42.569Z debug: [AskController] Request closed

**Reply by danny-avila** [reply]
I'm not sure how to help you with your setup. that does seem like the domain you've already included :/

**Reply by delta-whiplash** [reply]
okey I see if I have no worries am gonna search on my firewall

thanks for the fast answer

**Comment by delta-whiplash** [comment]
> Check your logs (in the ./logs directory of project root if using docker). Can't really help you much with these screenshots
am just trying the google models I don't want to pay for now
